service/service_test.go

```diff
@@ -18,10 +18,44 @@ import (
 	"context"
 	"testing"
 
+	"github.com/golang/mock/gomock"
 	"github.com/google/go-cmp/cmp"
 	pb "github.com/kubeflow/metadata/api"
+	mlmd "github.com/kubeflow/metadata/ml_metadata"
```

Comment:
With this package name, is Bazel build/test broken?

Reply:
No, they work fine. The presubmit test checks Bazel build and test.